INSTALL.md

```diff
@@ -20,7 +20,7 @@ much difficulty to most other Unix-like operating systems.
    npm.  In Terminal, run:
 
         $ sudo npm install npm -g
-        $ sudo np
+        $ sudo npm
```

Comment:
Actually I think we can just remove that line.  It looks like it was added accidentally.